pkg/tanka/workflow.go

```diff
@@ -50,8 +51,11 @@ func Apply(baseDir string, opts ApplyOpts) error {
 		// This is not fatal, the diff is not strictly required
 		log.Println("Error diffing:", err)
 	case diff == nil:
-		tmp := "Warning: There are no differences. Your apply may not do anything at all."
-		diff = &tmp
+		// If using KUBECTL_EXTERNAL_DIFF, the stdout buffer is always empty
+		if os.Getenv("KUBECTL_EXTERNAL_DIFF") == "" {
+			tmp := "Warning: There are no differences. Your apply may not do anything at all."
+			diff = &tmp
+		}
```

Comment:
hmm, can we keep this string? In our case, we use a `KUBECTL_EXTERNAL_DIFF` script to filter out some changes that are applied by other processes than Tanka (Flux CD adds some labels but we don't want to see them when applying manually)

Reply:
The issue here is that as was written, this message would be always displayed when `KUBECTL_EXTERNAL_DIFF` is set because Tanka is no longer capturing stdout from the diff utility.

I think the best way to support this use case is to have this controlled by a different environment variable than `KUBECTL_EXTERNAL_DIFF`. I have updated this so that you must set `KUBECTL_INTERACTIVE_DIFF=1` in order to get this behavior.